requirements-base.txt

```diff
@@ -54,6 +54,7 @@ redis-py-cluster>=2.1.0
 redis>=3.4.1
 requests-oauthlib>=1.2.0
 requests>=2.25.1
+requests-mock
```

Comment:
we should not be shipping mock dependencies to production

Reply:
Moved to requirements-dev

Reply:
we should be using `responses` instead -- it's maintained by us and already solves this problem